backend/fms_core/resources/experiment_run.py

```diff
@@ -0,0 +1,279 @@
+from django.core.exceptions import ValidationError
+from django.contrib.contenttypes.models import ContentType
+import reversion
+from import_export.fields import Field
+from import_export.widgets import ForeignKeyWidget, DateWidget
+from datetime import datetime
+from itertools import accumulate, islice
+import json
+
+from ..models import (
+    ExperimentRun,
+    Container,
+    Instrument,
+    ExperimentType,
+    Protocol,
+    Process,
+    ProcessMeasurement,
+    PropertyType,
+    PropertyValue,
+    Sample,
+    SampleLineage
+)
+from ._generic import GenericResource
+
+from ._utils import skip_rows, add_columns_to_preview, wipe_import_row_result
+from ..utils import (
+    blank_str_to_none,
+    float_to_decimal,
+    get_normalized_str,
+)
+
+from ..experiments import PROTOCOLS_BY_EXPERIMENT_TYPE_NAME
+
+__all__ = ["ExperimentRunResource"]
+
+
+process_content_type = ContentType.objects.get(app_label="fms_core", model="process")
```

Comment:
Why is this declared out of scope ? process_content_type is used once ... i would declare this just before usage.

Reply:
its in a loop (the one iterating through the experiment properties) i wanted to save some db call. I guess I could still put it at the top of the import_obj function though

Reply:
You are right we could save some hits to the db ... Maybe in the class init ? the import_obj will still get called many times ...

Reply:
I dont see the problem with it hanging there. It is not imported anywhere else. I am unsure about having it declared as a property/attribute of the ExperimentRunResource, but if you think it's still better, I wont argue :P

Reply:
I hate global variables mainly ... but yeah its not really a class attribute/property ... it's up to you but i would not define anything out of the scope of the class unless its a constant and even then...